frontstage/templates/surveys/help/surveys-help-my-survey-is-not-listed.html

```diff
@@ -39,9 +39,9 @@ <h1 class="ons-u-fs-xl">My survey is not listed</h1>
         You should <a href="{{ url_for('surveys_bp.add_survey') }}">add the new survey to your account</a> using the code in the letter.</p>
     <p>
         {% if is_survey_help_page is defined and  is_survey_help_page %}
-        <a href="{{ url_for('surveys_bp.send_help_message', option=option, sub_option='my-survey-is-not-listed') }}"> Send us a message</a>
+        <a href="{{ url_for('surveys_bp.send_help_message', option=option, sub_option='my-survey-is-not-listed', survey_name=survey_name, business_id=business_id, survey_id=survey_id, ce_id=ce_id) }}"> Send us a message</a>
```

Comment:
Ideally all of these should be absorbed into the backend, but I did so many it was beginning to get daft